src/main/features/response/routes/when-will-you-pay.ts

```diff
@@ -25,7 +25,7 @@ function formLabelFor (responseDraft: ResponseDraft): string {
   if (isAmountTooHighPartialResponse(responseDraft)) {
     return 'When will you pay the amount you admit you owe?'
   } else {
-    return 'When will you pay?'
+    return 'How do you want to pay?'
```

Comment:
please refactor the name of the files and the link in the paths file too

Reply:
I would prefer naming path, route and related files after properties of the draft so it is easier to find how draft property is bound.

In that case it would be `.../response/defendant-payment-option` and `defendant-payment-option.ts`

Reply:
Ok instead of using hardcoded values